### ISSUE TITLE
Keyboard navigation(Accessibility) doesn't work in a signature form.

### ISSUE THREAD

**Issue by wiadev** [hellosign/hellosign-embedded#144]
Hi
I'm using the hello-embedded iframe modal in my project and the keyboard navigation for Accessibility doesn't work in a signature form.

![Screen Shot 2020-02-16 at 7 01 32 PM](https://user-images.githubusercontent.com/18234060/74620710-e2755400-50ee-11ea-99e0-ec546452e911.png)

Could you take a look at it ASAP?

FYI, I've checked the form in hellosign.com and it's not even working over there.

Many thanks

**Comment by wiadev** [comment]
@nathanbuchar Could you please take a look at it?

**Comment by nathanbuchar** [comment]
Hi @wiadev, I'll forward to our team. By the way, HelloSign Embedded just wraps hellosign.com in an iFrame. That would explain why it wasn't working there either. At the moment, I'm uncertain of how much a11y support we have in our signer app, but I'll make sure someone follows up with you :)

**Comment by nathanbuchar** [comment]
As this is a missing feature on the HelloSign webapp and not HelloSign Embedded, I'll close this for now, but rest assured I've forwarded this to our webapp team!

**Comment by hazemhamedhs** [comment]
Hi there,

This is Hazem from API support stepping in for Nate. 

Please email us at apisupport@hellosign.com from the email address associated with your account and we’ll be able to update you on the status of this issue.

Thank you for your cooperation.
